src/conditionalOperators/Contains.test.ts

```diff
@@ -0,0 +1,16 @@
+import Contains from './Contains';
+
+describe('Contains operator', () => {
+  const operator = new Contains();
+  test('Output should be true', () => {
```

Comment:
should contain plutôt

faut pas faire un test sur la "valeur" attendue, mais sur le "comportement", n'oublie pas ça. C'est que du wording mais ça change la manière de comprendre les tests, et c'est primordial pour les corriger rapidement.